mne/decoding/transformer.py

```diff
@@ -235,14 +266,18 @@ def inverse_transform(self, X, y=None):
 
         Returns
         -------
-        epochs_data : array, shape (n_epochs, n_channels, n_times)
+        epochs : Instance of epochs
             The original data
         """
         if not isinstance(X, np.ndarray):
             raise ValueError("epochs_data should be of type ndarray (got %s)."
                              % type(X))
 
-        return X.reshape(-1, self.n_channels, self.n_times)
+        X_orig = X.reshape(-1, self.n_channels, self.n_times)
+        if not np.array_equal(self._epochs.get_data(), X_orig):
```

Comment:
Again, don't store the whole epochs, this can be very heavy; EpochsVectorizer isntances should be very light.